client/package.json

```diff
@@ -16,12 +16,18 @@
     "*.{js, jsx, css, scss}": "prettier --write"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^6.1.1",
```

Comment:
For icons, download the svg you want to use and import it
It reduces package count